### ISSUE TITLE
created_at does not match the Google Doc original

### ISSUE THREAD

**Issue by sparrell** [oasis-tcs/openc2-oc2ls#174]
This is a child of #143 

```
Page 8, Line 36:Â the example formatted property name (created_at) does not match the Google Doc original.

```

**Comment by sparrell** [comment]
Is this a formatting issue (eg font of "created_at") or a normative content issue (eg property should have different name)?

**Comment by dlemire60** [comment]
I believe it is a GDoc --> MD translation glitch.

GDoc reads "Property names are in bold style – created_at", where "created_at" is bold and shaded.

PDF reads "Property names are in bold style – **created_a**t", with no bold and everything after the first pair of asterisks is shaded. Jason and I have found a few other cases where the specifics of how formatting was applied in the GDoc led to odd presentations in MD (e.g., a trailing italicized space caused MD oddness).

**Comment by jmbrule** [comment]
I got the same issue in the SLPF.  Is a translation from googldoc to pdf glitch

**Comment by jmbrule** [comment]
This is NOT a translation glitch, was a misplaced asterisks.  I fixed it in the SLPF, easy fix

**Comment by romanojd** [comment]
I've addressed this issue within issue #143. Closing this comment.
